searchcore/src/vespa/searchcore/proton/server/searchview.cpp

```diff
@@ -96,7 +96,7 @@ convertLidsToGids(DocsumReply &reply, const DocsumRequest &request)
 DocsumReply::UP
 createEmptyReply(const DocsumRequest & request)
 {
-    DocsumReply::UP reply(new DocsumReply());
+    auto reply = std::make_unique<DocsumReply>();
```

Comment:
This is a pure style change.  Please don't mix this with functional changes.

Reply:
In #6627